### DISCUSSION TITLE
How to interpret rawinsonde data product?

### DISCUSSION THREAD

**Discussion by benghunt** [weather-gov/api#560] [Q&A] [answered]
Was unable to find any other NWS resource for this or details in the API,

but

[https://api.weather.gov/products/locations/{office}/types]

sometimes reveals Rawinsonde flights that can be accessed via the API, 

productCodes are MAN and SGL

their output looks like this

UMUS44 KLIX 040000
SGLLIX
72233 TTBB  54008 72233 00016 26410 11006 28640 22911 22638 
33776 13203 44759 12412 55748 12238 66725 10844 77702 10459 
88690 10058 99679 09661 11670 08860 22661 07649 33654 07250 
44623 05456 55612 05060 66603 04059 77592 03262 88588 02860 
99583 02656 11577 02262 22546 01358 33542 01756 44538 01957 
55530 02556 66526 02763 77518 03368 88512 04158 99506 04757 
11483 06361 22474 06961 33466 07956 44457 09556 55452 10141 
66449 09359 77446 09559 88418 13333 99396 16326 11361 21120 
22309 28744 33284 33356 44268 36734 55257 38950 66251 40166 
77244 41570 88235 43769 99232 44561 11226 46158 22222 46950 
33211 49756 44191 55750 55169 61557 66145 70156 77136 71156 
88133 69758 99125 72360 11120 72361 22116 73961 33110 73761 
31313 45208 82310 41414 824//=

PPBB  54008 72233 90023 00000 21013 20511 90467 20006 29003 
32507 9089/ 31007 28006 91245 28508 30011 33014 9169/ 32514 
30515 92015 32016 34013 00503 9304/ 02001 00000 94269 06511 
07026 05015 95023 08521 11516 14519=

how does one make sense of this data?

**Comment by dwhitemv25** [comment, marked as answer]
The format of the rawinsonde messages is defined in WMO Code Form FM 35 in the Manual on Codes Volume I.1, Part A, Section A. The Manual can be found here:

(https://library.wmo.int/doc_num.php?explnum_id=10235)

A practical example of coding for these messages can be found in NWS Manual 10-1401, Appendix F, found here:

(https://www.nws.noaa.gov/directives/sym/pd01014001curr.pdf)
